src/main/scala/decider/ProverStdIO.scala

```diff
@@ -347,6 +348,45 @@ abstract class ProverStdIO(uniqueId: String,
     toMap(stats)
   }
 
+  def statistics(): Map[String, String] = {
+    lastStatistics = queryStatistics()
+    lastStatistics
+  }
+
+  /**
```

Comment:
Could this delta computation be moved out of ProverStdIO? The statistics come from the prover, but their postprocessing is not tightly coupled to proving assertions.